catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/managers/ProductMgrImpl.java

```diff
@@ -119,10 +119,9 @@ public GetParametersRequest get_parameters(Product prod, String[] desired_parame
             getParamRequest = GetParametersRequestHelper.narrow(obj);
 
         } catch (WrongAdapter | WrongPolicy | UnsupportedEncodingException e) {
-            LOGGER.error("Unable to get ID from product reference: {}", NsilCorbaExceptionUtil.getExceptionDetails(e));
-            LOGGER.debug("Product reference exception details", e);
+            LOGGER.debug("Unable to get ID from product reference:", e);
```

Comment:
I didn't think it would be more useful than the entire stack trace. It is basically just parsing the entire stack and pulling things out. I believe that if you've already got the stack trace, it would be redundant.